README.md

```diff
@@ -77,19 +104,42 @@ Polling for a release that does not exist will return an empty release:
 }
 ```
 
+Because the output is JSON, you can pipe it through `jq` for fancier operations.
+
 ## Running the tests
 
-```
-go test ./...
+```bash
+go test ./... -v
 ```
 
 ## Design
 
-##### Defaults
+### Defaults
 
 | Parameter     | Description                     | Default  | Required  |
 | ------------- | ------------------------------- | -------- | --------- |
 | `--release`   | Name of the release to monitor  | `nil`    | True      |
 | `--timeout`   | Polling timeout in seconds      | `300`    | False     |
 | `--interval`  | Polling interval in seconds     | `5`      | False     |
 
+## What is Codacy
+
+[Codacy](https://www.codacy.com/) is an Automated Code Review Tool that monitors your technical debt, helps you improve your code quality, teaches best practices to your developers, and helps you save time in Code Reviews.
+
+### Among Codacy’s features
```

Comment:
I normally prefer to avoid these non-ASCII characters such as the quotation mark (it was probably copied from somewhere else, right?).

```suggestion
### Among Codacy's features
```

Reply:
20:20 vision! 
I think this was probably a smart quote copied from the text from Slack. fixing it 👍